site/docs/4.4/components/forms.md

```diff
@@ -1003,12 +970,18 @@ We recommend using client-side validation, but in case you require server-side v
 
 Validation styles are available for the following form controls and components:
 
-- `<input>`s and `<textarea>`s with `.form-control` (including up to one `.form-control` in input groups)
+- `<input>`s and `<textarea>`s with `.form-control`
 - `<select>`s with `.form-control` or `.custom-select`
 - `.form-check`s
 - `.custom-checkbox`s and `.custom-radio`s
 - `.custom-file`
 
+{% capture callout %}
+##### Input group validations
+For input groups, put feedback messages as a sibling element of the input group and add the `.is-{valid|invalid}` class to the `.input-group`. Placing feedback messages in a input group encounters border radius issues. [See this comment for workaround](https://github.com/twbs/bootstrap/issues/25110#issuecomment-586565165).
```

Comment:
Here's a suggestion on that word proofing. I'm okay linking to the comment since it's from one of us.

```suggestion
Input groups have difficulty with validation styles unfortunately. Our recommendation is to place feedback messages as sibling elements of the `.input-group` that has `.is-{valid|invalid}`. Placing feedback messages within input groups breaks the `border-radius`. [See this comment for workaround.](https://github.com/twbs/bootstrap/issues/25110#issuecomment-586565165)
```